wordpress-mobile/WordPress-iOS
# Issues/8062 consolidate keys

Closes #8062 
This PR consolidates the constants for various sets of keys and secrets used for Google Sign-up into a single set.  Individual build targets should update their own `.wpcom[_*]_app_credentials` files accordingly.

I tried this in the simulator with debug and internal build and everything seemed to work fine for me (mostly). 
I *was* getting an odd error compiling the internal build -- `gencredentials.rb` seemed to think it was encountering a nil `value` when parsing its credentials file. My copy of `.wpcom_internal_app_credentials` was identical to my copy of `.wpcom_app_credentials` with the excpetion of the values so I don't know what was going on there.  I added an nil check to the ruby script and that's sorted it.
I am still getting a warning in the alpha build about missing your custom google url scheme when tapping the google sign-in button. I've had zero luck sorting it ... to my eyes the code changes to the plist preprocessor looks correct and the internal build is *not* having this issue for me.  I'm tempted to call it a caching issue, but I also might just be blind to something I've mis-configured. Hoping another set of eyes can either verify it works or else spot the glitch.

To test:
Build and test debug, release, alpha, and internal builds. Confirm that:
- Builds are successful
- There are no crashes related to the affected constants after the change
- The google login feature works as expected

Summoning multiple sets of eyes for this one since I'm tinkering with unfamiliar things and one of the builds is being problematic, seemingly without rhyme or reason.

Needs review: @nheagy @elibud  and @diegoreymendez
